types/lib/model.d.ts

```diff
@@ -2846,10 +2846,11 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
 
 export type ModelType<TModelAttributes = any, TCreationAttributes = TModelAttributes> = new () => Model<TModelAttributes, TCreationAttributes>;
 
+type RemoveInstanceProperties<T> = { [K in keyof T]: T[K] };
 // Do not switch the order of `typeof Model` and `{ new(): M }`. For
 // instances created by `sequelize.define` to typecheck well, `typeof Model`
 // must come first for unknown reasons.
-export type ModelCtor<M extends Model> = typeof Model & { new(): M };
+export type ModelCtor<M extends Model> = RemoveInstanceProperties<typeof Model> & { new(): M };
```

Comment:
Another solution is:
```suggestion
export type ModelCtor<M extends Model> =Omit<typeof Model, never> & { new(): M };
```

I went with the approach of declaring a new type helper simply to add some semantics to what would otherwise be very counter intuitive.